<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFermentable</class>
 <widget class="QDialog" name="NewFermentable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Edit Fermentable</string>
  </property>
  <property name="windowIcon">
   <iconset resource="SB_Images.qrc">
    <normaloff>:/menu/sb2</normaloff>:/menu/sb2</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="8" colspan="2">
    <widget class="QCheckBox" name="fermentsCheck">
     <property name="text">
      <string>Ferments</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8" colspan="2">
    <widget class="QCheckBox" name="mashedCheck">
     <property name="text">
      <string>Mashed</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lovLabel">
     <property name="text">
      <string>Lovibond</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="costLabel">
     <property name="text">
      <string>Cost</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="3" colspan="8">
    <widget class="QPlainTextEdit" name="descriptionText"/>
   </item>
   <item row="3" column="8" colspan="2">
    <widget class="QCheckBox" name="steepedCheck">
     <property name="text">
      <string>Steeped</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QDoubleSpinBox" name="costSpin"/>
   </item>
   <item row="1" column="9">
    <widget class="QComboBox" name="stockUnits"/>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="stockLabel">
     <property name="text">
      <string>Stock</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7" colspan="2">
    <widget class="QDoubleSpinBox" name="stockSpin"/>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="lovSpin"/>
   </item>
   <item row="0" column="7">
    <widget class="QDoubleSpinBox" name="yieldSpin">
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="yieldLabel">
     <property name="text">
      <string>PPPG</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QComboBox" name="nameEdit">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="SB_Images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewFermentable</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewFermentable</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
